Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
It's hassle, but yeah, I can. A lot of the order was spat out automatically by `poet`'s love of alphabetisation, so the odd ones left at the top that weren't handled automatically by `poet` I slotted in in the same format.